gulpfile.js

```diff
@@ -9,15 +9,14 @@
 
 var gulp = require('gulp');
 
-var buildModules = __dirname + '/node_modules/fxos-build/node_modules/';
```

Comment:
Since we have our own bootstrap.js, I don't see why we need ```fxos-build``` at all - it looks like just an intermediate layer between our app and modules we need, but with more hassle and stale versions. app polyfil is just alameda (v0.2.0), I just use the latest alameda version directly (v1.0.0). And it will be much easier to update modules we use directly.

What do you think?